services/src/messaging/src/main/java/org/openkilda/messaging/command/flow/BidirectionalFlowRequest.java

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2017 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.command.flow;
+
+import org.openkilda.messaging.command.CommandData;
+import org.openkilda.messaging.payload.flow.FlowIdStatusPayload;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.Value;
+
+/**
+ * Represents get bidirectional flow northbound request.
+ */
+@JsonSerialize
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Value
+public class BidirectionalFlowRequest extends CommandData {
+    /**
+     * Serialization version number constant.
+     */
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * The request payload.
+     */
+    @JsonProperty("payload")
+    protected FlowIdStatusPayload payload;
```

Comment:
Why FlowIdStatusPayload? Not just flowId?

Reply:
Good question, because it just renamed version of FlowPathRequest with lombok annotations usage. Supposing refactor this in https://github.com/telstra/open-kilda/issues/1023 to

Reply:
... this is 1 field - just drop FlowIdStatusPayload from here and pass flowId. It will need one more change on CrudBolt side... no need to postpone it till #1023.